### ISSUE TITLE
"div() not a number" error when selecting token in send transaction screen 

### ISSUE THREAD

**Issue by lukaszfryc** [status-im/status-react#5139]
### Description

*Type*: Bug

*Summary*: If some collectible (like Cryptokitty) is selected in managed assets and user selects an asset in send transaction, the error appears. The app needs to be killed to work again.

<img src="https://user-images.githubusercontent.com/7532782/42580289-7714edba-852a-11e8-9ac0-323e1509229b.png" width="200">

### Reproduction

- Go to Wallet
- Open Manage Assets from options
- Select Cryptokitties
- Go back to Wallet
- Go to send transaction
- Tap on Asset to select different asset
-> Crash

### Additional Information
* Status version: 0.9.22
* Operating System: Android, iOS
* [Instabug report](https://dashboard.instabug.com/applications/status-6b2838c6-8064-445d-ae12-3c35e96f3a69/beta/bugs/384)

**Comment by trowacat** [comment]
Hi @lukaszfryc , I've investigated how to reproduce this error. In order for this to happen (on all devices I've tested so far) you must have selected one of the cryptocollectibles to be selected in your managed assets. (you don't need to have any, it just needs to show up on the wallet screen). Afterwards, this error comes up when selecting the type of currency to send. 

Once the collectible is removed from managed assets it works as intended again.
